.circleci/config.yml

```diff
@@ -59,6 +60,29 @@ jobs:
           name: DevOps checkout
           command: |
             git clone $DEVOPS_REPO $DEVOPS_WORKDIR
+      - run:
+          name: Start dev server
+          command: |
+            cp $DEVOPS_WORKDIR/environments/cypress .env
+            echo SOCKET_PORT=3001 >> .env
+            yarn dev || true
+          background: true
+      - run:
+          name: Wait for dev server to be healthy
+          command: curl -4 --retry 30 --retry-connrefused --retry-delay 10 http://localhost:3000/graphql
+      - run:
+          name: Run server tests
+          command: yarn workspace parabol-server test --runInBand --reporters=default --reporters=jest-junit
+          environment:
+            JEST_JUNIT_OUTPUT_DIR: ./reports/junit
+      - store_test_results:
+          path: ./packages/server/reports/junit/
+      - store_artifacts:
+          path: ./packages/server/reports/junit
```

Comment:
This configuration adds `jest` reporting to the CircleCI UI:

![Screen Shot 2022-02-01 at 11 52 01 AM](https://user-images.githubusercontent.com/630449/152032158-16523e2c-8df2-41a7-9945-836c5eea5a26.png)

This provides a few distinct features:

1. Instead of digging through log output, failing tests are easy to find via the UI.
2. CircleCI will monitor tests that fail via their [test insights](https://circleci.com/docs/2.0/insights-tests/) feature. This helps identify flaky tests and more.

Reply:
+2 Nice!